ipatests/test_integration/test_hsm.py

```diff
@@ -0,0 +1,1361 @@
+#
```

Comment:
I think these are all covered. I put some pretty big timeouts in some of them. I'm hoping to be able to tell the basic runtime once we run the tests so we can tone it down a bit. It will need to cover both the softhsm and hardware HSMs which can be slower with some accesses.